### ISSUE TITLE
I have a few questions about the paper.

### ISSUE THREAD

**Issue by SeongYeonPark** [OlaWod/FreeVC#2]
Hi, thank you for such creative work on Voice Conversion.

I have 3 questions about your paper, FreeVC: Towards High-Quality Text-Free One-Shot Voice Conversion.
It would help a lot if you would answer them.

1. Did you freeze the WavLM module during training? (Also, did you use pre-trained weights for this module?)
2. Can I get some architectural information about the bottleneck extractor? Is it just two fully connected layers that map the x_ssl features into d and then 2d dimension?
3. During training, using the SR augmentation horizontally would un-align the source mel and the target wav.
So I assume that you only used the SR augmentation vertically during training. Is this true?

++ Edit
I have one further question.
Could you possibly share the config file for the Hifi-gan that vocodes the augmented x_mel into WavLm input y' s?

**Comment by OlaWod** [comment]
Thank your for your interest in our work.

1. We freeze the WavLM module and use the pre-trained weights.
2. The bottleneck extractor consists of a linear projection layer that projects the WavLM feature into $d$-dim hidden representation, 16 layers of non-causal WaveNet residual blocks, and a linear projection layer that projects $d$-dim hidden representation into $2d$-dim hidden representation, which is latter split into $d$-dim $\mu_{\theta}$ and $d$-dim $\sigma_{\theta}$.
3. Yes.
4. We use the same [config](https://github.com/jik876/hifi-gan/blob/master/config_v1.json) as the official HiFi-GAN for producing $y'$.

**Comment by SeongYeonPark** [comment]
Thank you very much for your clear answer.

So it can be said that
1. The mel-spectrograms that are input to the SR-augmentation part are obtained with 22050Hz waves, using hop size 256, window size 1024 (following the config v1 of the official HiFi-GAN repository)
2. and the linear-spectrograms that are input to the posterior encoder are obtained with 16000Hz waves, using hop size 320, window size 1280 (as said in your paper, section 3.1)
3. Perhaps the waveform reconstructed after SR-augmentation and HiFi-GAN config v1 model (which will have a sampling rate 22050 Hz) is resampled to 16000Hz before inputting into WavLM?

**Comment by OlaWod** [comment]
> Thank you very much for your clear answer.
> 
> So it can be said that
> 
> 1. The mel-spectrograms that are input to the SR-augmentation part are obtained with 22050Hz waves, using hop size 256, window size 1024 (following the config v1 of the official HiFi-GAN repository)
> 2. and the linear-spectrograms that are input to the posterior encoder are obtained with 16000Hz waves, using hop size 320, window size 1280 (as said in your paper, section 3.1)
> 3. Perhaps the waveform reconstructed after SR-augmentation and HiFi-GAN config v1 model (which will have a sampling rate 22050 Hz) is resampled to 16000Hz before inputting into WavLM?

Yes.
